features/step_definitions/sign_up_steps.rb

```diff
@@ -0,0 +1,19 @@
+Given(/^I visit the index page$/) do
+  visit root_path
+end
+
+Given(/^I click "([^"]*)"$/) do |arg1|
+  click_link_or_button arg1
+end
+
+Then(/^I fill in "([^"]*)" with "([^"]*)"$/) do |title, content|
+  fill_in title, with: content
+end
```

Comment:
These three steps are also basic steps. Sign up steps should only have steps that are specific to signing up. Clicking buttons and being on the index page are *not* things that only happen when you sign up.